### PR TITLE
Fix remove unused function and whitespaces

### PR DESCRIPTION
**Pull request by heavenshell**
Cosmetic changes 😉

### CODE REVIEW

File: autoload/lsp_settings/ui.vim [hunk 1]
```diff
@@ -71,14 +71,11 @@ function! s:uninstall() abort
   exe 'LspUninstallServer' l:command
 endfunction
 
-function! s:update() abort
-endfunction
-
```
Comment (heavenshell) [inline autoload/lsp_settings/ui.vim:76]: unused?

File: autoload/lsp_settings/ui.vim [hunk 2]
```diff
@@ -71,14 +71,11 @@ function! s:uninstall() abort
...
 function! lsp_settings#ui#open() abort
   silent new __LSP_SETTINGS__
   only!
   setlocal buftype=nofile bufhidden=wipe noswapfile cursorline
-
```
Comment (heavenshell) [inline autoload/lsp_settings/ui.vim:81]: whitespace

File: autoload/lsp_settings/ui.vim [hunk 3]
```diff
@@ -92,7 +89,7 @@ function! lsp_settings#ui#open() abort
           let l:missing = 1
           break
         endif
-      endfor
```
Comment (heavenshell) [inline autoload/lsp_settings/ui.vim:95]: whitespaces

File: autoload/lsp_settings/ui.vim [hunk 4]
```diff
@@ -102,7 +99,7 @@ function! lsp_settings#ui#open() abort
       else
         call add(l:types[l:conf.command], l:ft)
       endif
-    endfor
```
Comment (heavenshell) [inline autoload/lsp_settings/ui.vim:105]: whitespaces

**Comment by mattn** [comment]
Thanks!

**Pull request merged**